packages/fxa-auth-server/scripts/recorded-future/check-and-reset.ts

```diff
@@ -156,13 +161,52 @@ Limit: ${resultsPerPageLimit}
     }
   }
 
-  if (accountEmails.length === 0) {
+  if (accountLogins.length === 0) {
     console.log('No account emails found.');
     return 0;
   }
 
+  const credentialsLookupFn = createCredentialsLookupFn(client);
+  const cleartextCredentials = await credentialsLookupFn(accountLogins, {
+    first_downloaded_gte: firstDownloadedDateIsoString,
+  });
+  statsd.increment(
+    'recorded-future.identity.credentials-lookup.cleartext-total',
+    cleartextCredentials.length
+  );
+
+  const verifyPassword = createVerifyPasswordFn(
+    Password,
+    authDb.checkPassword.bind(authDb)
+  );
+
+  const defaultConcurrency = 100;
```

Comment:
Take this worth a grain of salt. But maybe make this an option?

Reply:
I'm not opposed to it.  I just don't foresee us fiddling with it.  I will add it in the next PR for this script.

Reply:
Since I'm going to make other changes, I'll add it in this PR.